FV3/testsuite/data/c48_basic/test_baseline/input.nml

```diff
@@ -42,6 +43,7 @@
  npz    = 63
  make_nh = .T.
  fv_debug = .F.
+ disable_fv_restart_write = .F.
  fv_testsuite_output = .T.
```

Comment:
```suggestion
 fv_testsuite_output = .T.
```

Reply:
Same as above.  Just added the option so a new user/developer can see that it is an available option.  Default value of .false. ensures that no change is made to the default output behavior.